### ISSUE TITLE
Error: unsupported on mqtt.connect

### ISSUE THREAD

**Issue by Viburnum** [iobroker-community-adapters/ioBroker.worx#583]
**Describe the bug**  
Maybe an issue with newer node versions and deprecated ciphers provided by landroid? See https://github.com/nodejs/node/issues/40672

**To Reproduce**  
1. Update to Github Version
2. Reenter Password
3. Start

**Screenshots & Logfiles**  
```
2022-10-04 08:27:07.014 - error: worx.0 (29271) Error: Could not check object existence of worx.0.201930198403023376AE.rawMqtt.features.chassis: Cannot check object existence of "worx.0.201930198403023376AE.rawMqtt.features.chassis": Connection is closed.
2022-10-04 08:27:07.089 - error: host.raspberrypi Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2022-10-04 08:27:07.090 - error: host.raspberrypi Caught by controller[1]: Error: unsupported
2022-10-04 08:27:07.090 - error: host.raspberrypi Caught by controller[1]: at configSecureContext (node:internal/tls/secure-context:279:15)
2022-10-04 08:27:07.090 - error: host.raspberrypi Caught by controller[1]: at Object.createSecureContext (node:_tls_common:117:3)
2022-10-04 08:27:07.091 - error: host.raspberrypi Caught by controller[1]: at Object.connect (node:_tls_wrap:1629:48)
2022-10-04 08:27:07.091 - error: host.raspberrypi Caught by controller[1]: at Object.buildBuilder (/opt/iobroker/node_modules/mqtt/lib/connect/tls.js:20:26)
2022-10-04 08:27:07.091 - error: host.raspberrypi Caught by controller[1]: at MqttClient.wrapper [as streamBuilder] (/opt/iobroker/node_modules/mqtt/lib/connect/index.js:155:36)
2022-10-04 08:27:07.091 - error: host.raspberrypi Caught by controller[1]: at MqttClient._setupStream (/opt/iobroker/node_modules/mqtt/lib/client.js:415:22)
2022-10-04 08:27:07.091 - error: host.raspberrypi Caught by controller[1]: at new MqttClient (/opt/iobroker/node_modules/mqtt/lib/client.js:395:8)
2022-10-04 08:27:07.091 - error: host.raspberrypi Caught by controller[1]: at Object.connect (/opt/iobroker/node_modules/mqtt/lib/connect/index.js:157:18)
2022-10-04 08:27:07.092 - error: host.raspberrypi Caught by controller[1]: at Worx.connectMqtt (/opt/iobroker/node_modules/iobroker.worx/main.js:655:27)
2022-10-04 08:27:07.092 - error: host.raspberrypi Caught by controller[1]: at Worx.start_mqtt (/opt/iobroker/node_modules/iobroker.worx/main.js:638:18)
2022-10-04 08:27:07.092 - error: host.raspberrypi instance system.adapter.worx.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
```

**Versions:**
Adapter version: v2.0.0 - Github Version
JS-Controller version: 4.0.23
Node version: v18.9.0
Operating system: Raspbian GNU/Linux 11 (bullseye)
Landroid WR141E

**Comment by TA2k** [comment]
It is an issue for >= Node v17.1 I will look into this.
current GitHub version will fetch data but not via Mqtt and not able to send commands

**Comment by TA2k** [comment]
fixed with v2.0.1
